rosdep/python.yaml

```diff
@@ -5749,6 +5749,16 @@ quadprog-pip:
     pip: [quadprog]
   ubuntu:
     pip: [quadprog]
+rafcon-pip:
```

Comment:
Can you please rename (and therefore relocate) this key to `python3-rafcon-pip`. This isn't currently specified as a requirement but it's being added to the contribution guidelines in #23836.

Reply:
Done!